__tests__/steps/setStepsMap.spec.ts

```diff
@@ -85,5 +86,12 @@ describe('setSteps', () => {
         },
       });
     });
+
+    it('returns when no steps are provided', () => { 
```

Comment:
`setSteps returns when no steps are provided`, what it returns? you are not even calling `setSteps` in this test.